tests/snapshot-gating.fmf

```diff
@@ -36,6 +36,9 @@ prepare:
       # already in the system.
       - dnf -y install --best llvm-libs
 
+      # Use a newer llvm-test-suite version from copr.
```

Comment:
This change makes sense to me. Just a little comment: could you remove the blank line above this one so we have the whole section packed?

Reply:
Done!